peps/pep-0740.rst

```diff
@@ -203,25 +203,35 @@ The following changes are made to the
 :ref:`simple repository API <packaging:simple-repository-api-base>`:
 
 * When an uploaded file has one or more attestations, the index **MAY**
-  provide a ``.provenance`` file adjacent to the hosted distribution.
-  The format of the ``.provenance`` file **SHALL** be a JSON-encoded
-  :ref:`provenance object <provenance-object>`, which **SHALL** contain
-  the file's attestations.
+  provide a provenance file associated with the hosted distribution
+  containing those attestations. The format of the provenance file
```

Comment:
This makes it seem like the hosted distribution contains the attestations, but you're actually referring to the provenance file. Maybe:

```suggestion
  provide a provenance file containing attestations associated with
  a given distribution. The format of the provenance file
```

Reply:
That's much better, thanks!